css-scoping-1/css-scoping-shadow-assigned-node-with-before-after.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>CSS Scoping Module Level 1 - :before and :after pseudo class' contents on a node assigned to a slot element must be rendered</title>
```

Comment:
https://drafts.csswg.org/css-pseudo-4/#selectordef-before
`:before` and `:after` should be `::before` and `::after` as they are pseudo elements.